commons-rdf-simple/src/main/java/org/apache/commons/rdf/simple/experimental/AbstractRDFParser.java

```diff
@@ -200,19 +200,19 @@
      *         been set
      */
     public Optional<IRI> getSourceIri() {
-        return sourceIri;
+        return Optional.ofNullable(sourceIri);
     }
 
-    private Optional<RDF> rdfTermFactory = Optional.empty();
-    private Optional<RDFSyntax> contentTypeSyntax = Optional.empty();
-    private Optional<String> contentType = Optional.empty();
-    private Optional<IRI> base = Optional.empty();
-    private Optional<InputStream> sourceInputStream = Optional.empty();
-    private Optional<Path> sourceFile = Optional.empty();
-    private Optional<IRI> sourceIri = Optional.empty();
+    private RDF rdfTermFactory = null;
+    private RDFSyntax contentTypeSyntax = null;
+    private String contentType = null;
+    private IRI base = null;
+    private InputStream sourceInputStream = null;
```

Comment:
This may be a sticking point for actually enabling ``Serializable``, but the changes to not use optional field values look good to me.